### ISSUE TITLE
Publish non-alpha to hex?

### ISSUE THREAD

**Issue by maxbeizer** [zotonic/eiconv#4]
First off, thank you to all who worked on/maintain this ❤️ .

Related issue: https://github.com/mailman-elixir/mailman/issues/78#issuecomment-344860752

Hex does not allow users to push packages with alpha dependencies. It returns the following error:

```
** (Mix) A stable package release cannot have a pre-release dependency
```

Is there any reason the version pushed to hex is an alpha? Is there any reason not to bump to 1.0.0? Doing so would allow other packages to use the hex version as a dependency.

cc @ddeboer 
also related: https://github.com/zotonic/eiconv/issues/3

**Comment by mworrell** [comment]
I don’t see any problems with making this a 1.0.0.

@ddeboer can you bump the version number?

**Comment by ddeboer** [comment]
Published 1.0.0!

**Comment by maxbeizer** [comment]
thanks @mworrell @ddeboer !!
